ibc-core/ics24-host/types/src/error.rs

```diff
@@ -16,6 +16,8 @@ pub enum IdentifierError {
     RevisionNumberOverflow,
     /// String `{value}` cannot be converted to packet sequence, error: `{reason}`
     InvalidStringAsSequence { value: String, reason: String },
+    /// String `{id}` does not start with `{name}`
+    InvalidNamedIndex { id: String, name: String },
```

Comment:
There is an `InvalidPrefix` variant as well, which I see not used anywhere. It appears meant to serve the same purpose. Can't use that or merge these two?

Reply:
dfe5a7c